src/json_reporter.cc

```diff
@@ -254,9 +254,14 @@ void JSONReporter::PrintRunData(Run const& run) {
       BENCHMARK_UNREACHABLE();
     }()) << ",\n";
   }
-  if (run.error_occurred) {
-    out << indent << FormatKV("error_occurred", run.error_occurred) << ",\n";
-    out << indent << FormatKV("error_message", run.error_message) << ",\n";
+  if (run.skipped) {
+    if (run.skip_is_error) {
```

Comment:
this could be a if..else if.. statement to remove the nesting if's but i feel like the logic is easier to follow this way